benchmark/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM --platform=linux/amd64 debian:buster-slim
```

Comment:
Can/should this Dockerfile be scoped to the sirun directory?

Reply:
Good point, I'll move the Dockerfile into the sirun directory.

The build commands will still need to be run from within the project root, however.